marshmallow_dataclass/__init__.py

```diff
@@ -34,8 +34,9 @@ class User:
       })
       Schema: ClassVar[Type[Schema]] = Schema # For the type checker
 """
-
+from enum import Enum, EnumMeta
 import dataclasses
+import inspect
```

Comment:
Do we still need `inspect` ?

Reply:
I think the diffs are confusing because I inadvertently moved this import up a few lines. It's actually still used elsewhere in the `class_schema` function. 

Do you have a preferred order for imports? My instinct is normally a) python standard library, b) 3rd party packages, c) imports from within the package itself. Happy to change this or leave it alone!

Reply:
Oh, you are right, sorry !